src/it/settings.xml

```diff
@@ -36,6 +36,9 @@ under the License.
       <activation>
         <activeByDefault>true</activeByDefault>
       </activation>
+      <properties>
+        <disableXmlReport>true</disableXmlReport>
```

Comment:
This is the part that makes me somewhat uncomfortable.

Reply:
well you wanted to avoid noise (https://github.com/jenkinsci/plugin-pom/pull/508#pullrequestreview-891287591) from those files which should not be scanned by `buildPlugin` so I tried to address this in 2 ways: this and this one https://github.com/jenkins-infra/pipeline-library/pull/307
and this would have impact only if we have an IT test running `mvn site` and making some assertions on having the surefire-report html file correctly generated. 
surefire does not use those files to detect error there are only here to build reports by other plugins.

Reply:
> surefire does not use those files to detect error there are only here to build reports by other plugins.

Yes, I know.